source/dashboard.rst

```diff
@@ -62,3 +62,48 @@ Clone Github repository
 -----------------------
 
 See :doc:`github`.
+
+Organizing your repositories with categories
+--------------------------------------------
+
+Over time, the list of repositories you deal with can grow long.  As it does, it can become difficult to
```

Comment:
Please use one space after period

Reply:
Okay!  Done throughout the section in subsequent commit.